english/lunge-for-the-lighter/steep-slope/steep-slope.md

```diff
@@ -4,4 +4,7 @@ situation reeks of deception, and you have a vague recollection that you've seen
 this opening before...while being chased by marshmallow men who were shooting
 lasers. As you approach the opening, you see that the slope is much steeper than
 it first appeared, and you can't see where it leads. Just then, you notice a
-big red button above the opening.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.